### PR TITLE
Import Teams

### PR DESCRIPTION
**Pull request by Apexal**
This PR refactors the import pages to reuse the templates, and introduces a CSV import for Submitty teams. It creates/updates users, enrollments, and projects. With this, we can get an accurate model of all projects and their teams for this semester!

Unfortunately, people submitted slightly different project names on Submitty versus on the original Google Form I used to import. Therefore, we might end up with duplicate projects like "open circuits" and "OpenCircuits". I will run this in prod once merged and manually fix these issues.

### CODE REVIEW

File: portal/templates/portal/admin/import/import.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+                <div class="content">
+                    <p>The CSV should be exported from {{ source }} and is expected to contain the following columns (order doesn't matter, exact name does):</p>
+                    <ul>
+                        {% for col in expected_columns %}
+                        <li><code>{{ col }}</code></li>
+                        {% endfor %}
+                    </ul>
+                    <p>
+                        Importing is <strong>imdepotent</strong>, so you can upload the same file (or the same file with later additions)
```
Comment (bnidevs) [inline portal/templates/portal/admin/import/import.html:22]: can you replace with something from https://thesaurus.com

**Review by bnidevs — APPROVED** [review]

File: portal/templates/portal/admin/import/import.html (hunk 1 above)
Comment (Apexal) [reply portal/templates/portal/admin/import/import.html:22]: what

Comment (bnidevs) [reply portal/templates/portal/admin/import/import.html:22]: imdepotent is too hard a word, pick something easier

Comment (Apexal) [reply portal/templates/portal/admin/import/import.html:22]: My reasoning including it here is that it's the correct, technical term, and I immediately define it in the rest of the sentence. Is it not clear that it is being defined? I think there's value in teaching the word here, what do you think? I'm not that attached to it. I'll merge in the meantime.

**Pull request merged**